pybuildkite/pipelines.py

```diff
@@ -34,3 +34,28 @@ def get_pipeline(self, organization, pipeline_name):
         :return: A pipeline
         """
         return self.client.get(self.path.format(organization) + pipeline_name)
+
+    def create_pipeline(self, organization, pipeline_name, git_repository, commands_to_run):
+        """
+        Create a pipeline
+        :param organization: Organization slug
+        :param pipeline_name:Pipeline slug
+        :param git_repository: repo URL
+        :param commands_to_run: build pipeline steps
+        :return:
+        """
+        data = {'name': pipeline_name,
+                'repository': git_repository,
+                'steps': [dict(type='script', name=':pipeline:', command=commands_to_run)]}
```

Comment:
It looks like we're being really prescriptive about the steps we're allowing. I feel like we should just take in a list of `dicts`. If the API requires at least one, I'd say we set a default to maybe `buildkite-agent pipeline upload` as the command.

Thoughts?

Reply:
I agree with that.As default,we could have a default
`[
        {
            "type": "script",
            "name": ":pipeline:",
            "command": "buildkite-agent pipeline upload",

        }
    ]` and then in PATCH request,the user would add other scripts if needed.
 If we make the API take a list of build steps then the parameter list would be long.
https://buildkite.com/docs/apis/rest-api/pipelines#create-a-pipeline

Reply:
Yeah maybe that default you responded could be the default param in the actual method. Then the user could override it with anything they’d want. And we could put some documentation of how it could be used in the method docstring?

Reply:
Yes, that makes it much better.